client/backend.js

```diff
@@ -212,5 +212,17 @@ module.exports = {
             success: cb,
             error: xhrError
         });
+    },
+    
+    setTypeForOperation: function(operationId, operationTypeId, cb) {
```

Comment:
new backend functions should use the Promise model, but it's fine to let as is; I plan doing a full pass at some point anyways.